pkg/network/ebpf/c/runtime/conntrack-types.h

```diff
@@ -3,6 +3,23 @@
 
 #include <linux/types.h>
 
+typedef struct {
+    /* Using the type unsigned __int128 generates an error in the ebpf verifier */
+    __u64 saddr_h;
+    __u64 saddr_l;
+    __u64 daddr_h;
+    __u64 daddr_l;
+    __u16 sport;
+    __u16 dport;
+    __u32 netns;
+    // Metadata description:
+    // First bit indicates if the connection is TCP (1) or UDP (0)
+    // Second bit indicates if the connection is V6 (1) or V4 (0)
+    __u32 metadata; // This is that big because it seems that we atleast need a 32-bit aligned struct
+
+    __u32 _pad;
+} conntrack_tuple_t;
```

Comment:
They are intentionally different because the conntrack version does not have a `netns`.

Reply:
`netns` is currently a field of both structs - are you saying it's not supposed to be in this one?

If that's the case - does removing `netns` as a field from this struct make a big enough difference to warrant having 2 distinct types? Since we were previously able to use `conn_tuple_t` in the places you're proposing we now use `conntrack_tuple_t`, I'm just having a hard time understanding why the new type is necessary

Reply:
Sorry, I meant the `pid` field is not in this tuple.

Reply:
The conntrack code is completely separate from the rest of the tracer in terms of eBPF, so there is no need for cross-compatibility.

Reply:
I kept them separate because the conntrack code has no way of populating the PID field at the moment. This way you cannot accidentally pass a conntrack tuple to a function expecting the PID field to be present and correct.